figures/mapping-histology-labels.Rmd

```diff
@@ -140,15 +125,12 @@ lgat <- c("other astrocytic tumor", "low-grade astrocytic tumor")
 
 ```{r}
 histology_table <- working_metadata %>% 
+  dplyr::filter(sample_type == "Tumor") %>%
   dplyr::mutate(
     # NAs are really Normals
```

Comment:
```suggestion
    # NAs are tumor samples  with pathology_diagnosis ==`Dysplasia/Gliosis` and `Other`
```

Since you've added the sample_type=="Tumor" above I think we should update the comment here

```
 metadata %>% dplyr::filter(is.na(cancer_group),sample_type == "Tumor") %>% dplyr::group_by(cohort,pathology_diagnosis,sample_type) %>% dplyr::tally()
# A tibble: 2 × 4
# Groups:   cohort, pathology_diagnosis [2]
  cohort pathology_diagnosis sample_type     n
  <chr>  <chr>               <chr>       <int>
1 PBTA   Dysplasia/Gliosis   Tumor          27
2 PBTA   Other               Tumor          37
```

The above pathology_diagnosis were removed from cancer_group as per https://github.com/PediatricOpenTargets/ticket-tracker/issues/48#issuecomment-867895997 so I think these should removed from the label/color list below as well. 

**To-Do:**  We will need an update in the various plotting scripts to remove any sample with cancer_group== `NA` as well so that it doesn't error out. 
 cc @jharenza for confirmation.